src/main/cpp/processors/filter.cpp

```diff
@@ -29,26 +29,21 @@
  * @param aResonance {float} resonance
  * @param aMinFreq {float} minimum cutoff frequency in Hz, required for LFO automation
  * @param aMaxFreq {float} maximum cutoff frequency in Hz, required for LFO automation
- * @param aLfoRate {float} LFO speed in Hz, 0 for OFF
  * @param numChannels {int} amount of output channels
```

Comment:
Filter should not be bothered by knowing its LFO speed in the constructor (especially funny as Filter could not construct LFO's on its own..).